src/components/PercentualDifference/percentualDifference.tsx

```diff
@@ -0,0 +1,39 @@
+export class PercentualDifference {
+  firstValue: number
+  lastValue: number
+  theme: { green1: string; red1: string; gray1: string }
+
+  constructor(firstValue: string, lastValue: string, theme: { green1: string; red1: string; gray1: string }) {
+    this.firstValue = +firstValue
+    this.lastValue = +lastValue
```

Comment:
firstValue and lastValue should be typed as number

```suggestion
  constructor(firstValue: number, lastValue: number, theme: { green1: string; red1: string; gray1: string }) {
    this.firstValue = firstValue
    this.lastValue = lastValue
```

Reply:
Fixed.